src/tools/rustdoc-gui-test/src/main.rs

```diff
@@ -128,7 +128,10 @@ If you want to install the `browser-ui-test` dependency, run `npm install browse
                 }
             }
 
-            try_run(&mut cargo, config.verbose);
+            if !try_run(&mut cargo, config.verbose) {
```

Comment:
Looking at cause if this bug, i want to slap `must_use` over `try_run` and find all similar places.

Reply:
That's a good idea. Doing that shortly.